dsp/modules/gpt3.py

```diff
@@ -1,34 +1,44 @@
+import copy
 import functools
+import importlib.util
 import json
 import logging
 from typing import Any, Literal, Optional, cast
+
 import backoff
 import httpx
-try:
+
+from dsp.modules.cache_utils import CacheMemory, NotebookCacheMemory, cache_turn_on
+from dsp.modules.lm import LM
+from dsp.utils.settings import settings
+
+
+def is_module_installed(module_name):
```

Comment:
just curious why this is needed. does the try/except block not take care of this check if langfuse is not supported?